cli/config_file.rs

```diff
@@ -266,10 +266,34 @@ impl Serialize for TsConfig {
   }
 }
 
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default)]
+pub struct LintRulesConfig {
+  pub tags: Option<Vec<String>>,
+  pub include: Option<Vec<String>>,
+  pub exclude: Option<Vec<String>>,
```

Comment:
I don't remember this `exclude` being in the first draft we discussed. I have reservations about this.

Reply:
I believe Ryan didn't add it to https://github.com/denoland/deno/issues/11686 but it was based on existing config in `dlint`. What are your reservations?

Reply:
I'll just give up on my argument against this. We discussed it so often earlier this year (registry wants to do linting to score packages, can't do that reliably if people start disabling rules in the recommended set).

Reply:
Can't do this reliably if you specify additional rules as well. We can still discuss it, no need to give up